core/src/main/java/com/amplifyframework/predictions/models/FaceLivenessChallengeType.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2024 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.amplifyframework.predictions.models
+
+import com.amplifyframework.annotations.InternalAmplifyApi
+
+@InternalAmplifyApi
+enum class FaceLivenessChallengeType {
```

Comment:
Do we need to duplicate ChallengeType and FaceLivenessChallengeType. They are essentially the same.

Reply:
For some reason I had this assumption that we needed to keep server/client models separate but yes they are exactly the same. I can keep this core FaceLivenessChallengeType and reference this from the server model and remove `ChallengeType`